content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,248 @@
+parameters, go to :menuselection:`Payroll --> Configuration --> Rule Parameters`.
+
+Other Input Types
+-----------------
+
+When creating payslips, it is sometimes necessary to add other entries for specific circumstances,
+like expenses, reimbursements, or deductions. These other inputs can be seen by going to
+:menuselection:`Payroll --> Configuration --> Other Input Types`.
+
+.. image:: configuration/other-input.png
+   :align: center
+   :alt: Other input types for payroll.
+
+A new *Input Type* can be created by clicking the :guilabel:`Create` smart button. Enter the
+description, the code, and which structure type it applies to. Click the :guilabel:`Save` smart
+button when donesmart button to save the changes, or :guilabel:`Discard` to delete the entry.
+
+.. image:: configuration/input-type-new.png
+   :align: center
+   :alt: Create a new Input Type.
+
+Salary Package Configurator
+===========================
+
+Advantages
+----------
+
+When offering potential employees a position, there can be certain **advantages** to make the offer
+more appealing. Any specific advantage is listed in the advantages section of the configuration
+menu. To see the advantages, go to  :menuselection:`Payroll --> Configuration --> Advantages`.
+
+.. image:: configuration/advantages.png
+   :align: center
+   :alt: Settings available for payroll.
+
+To make a new advantage, click the :guilabel:`Create` smart button. A pop up appears for the
+advantage. Enter the information in the fields, then click the :guilabel:`Save` smart button to save
+the changes, or :guilabel:`Discard` to delete the entry.
+
+.. image:: configuration/new-advantage.png
+   :align: center
+   :alt: List of advantages employee's can have.
+
+
+Personal Info
+-------------
+
+The *Personal Information* section lists all of the fields that are available to enter on the
+employee's card.
+
+.. image:: configuration/personal-info.png
+   :align: center
+   :alt: Personal information that appear on employee cards to enter.
+
+To edit an entry, click on the line, and a box with the configuration for that piece of information
+will appear. To edit the information, click the :guilabel:`Edit` smart button, and modify the entry.
+When done, click :guilabel:`Save` or :guilabel:`Discard` to save the information or cancel the
+edits.
+
+To create a new entry, click the :guilabel:`Create` smart button. A box will appear with all of the
+fields to enter.
+
+.. image:: configuration/personal-new.png
+   :align: center
+   :alt: New personal information entry.
+
+The two most important fields are the *Is Required* checkbox, and the *Display Type* drop down
+selection. Checking the *Is Required* box will make the field mandatory on the Employee's card.
+
+The *Display Type* drop down allows for the information to be entered in a variety of ways, from a
+text box, a customizable radio button, a check box, a document, and more. This will affect how the
+field appears.
```

Comment:
Fixing formatting.

Also, for the last line, "This will affect how the field appears", where does it appear?
```suggestion
The two most important fields are :guilabel:`Is Required` and :guilabel:`Display Type`. Checking the :guilabel:`Is Required` box will make the field mandatory on the Employee's card.

The :guilabel:`Display Type` drop-down menu allows for the information to be entered in a variety of ways, from a
text box, a customizable radio button, a check box, a document, and more. This will affect how the
field appears.
```